### DISCUSSION TITLE
error ejecutando el comando mvn verify

### DISCUSSION THREAD

**Discussion by JuanMartinIwassjuk** [dds-jv/foro#354] [Consultas de herramientas] [answered]
Holaa, estaba por entregar el tp de macowins, el mvn test me corre sin problemas pero cuando ejecuto el mvn verify me tira un par de bugs, ocurre cuando verifica  los code smells (creo).  
![bugmaven](https://user-images.githubusercontent.com/81983051/165194174-98482de5-2b34-4b2e-9c2a-c508771082c8.png)

**Comment by JuanMartinIwassjuk** [comment, marked as answer]
> Holaa, estaba por entregar el tp de macowins, el mvn test me corre sin problemas pero cuando ejecuto el mvn verify me tira un par de bugs, ocurre cuando verifica los code smells (creo). ![bugmaven](https://user-images.githubusercontent.com/81983051/165194174-98482de5-2b34-4b2e-9c2a-c508771082c8.png)
dejo el texto de la consola por las dudas:
INFO] --- spotbugs-maven-plugin:4.2.0:check (Validar Code Smells) @ ejercicio ---
[INFO] BugInstance size is 2
[INFO] Error size is 0
[INFO] Total bugs: 2
[ERROR] Medium: Unread field: ar.edu.utn.frba.dds.macowins.Venta.fecha [ar.edu.utn.frba.dds.macowins.Venta] At macowins.java:[line 25] URF_UNREAD_FIELD
[ERROR] Medium: The class name ar.edu.utn.frba.dds.macowins.macowins doesn't start with an upper case letter [ar.edu.utn.frba.dds.macowins.macowins] At macowins.java:[lines 7-12] NM_CLASS_NAMING_CONVENTION
[INFO] 


To see bug detail using the Spotbugs GUI, use the following command "mvn spotbugs:gui"



[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  15.554 s
[INFO] Finished at: 2022-04-25T21:27:42-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.spotbugs:spotbugs-maven-plugin:4.2.0:check (Validar Code Smells) on project ejercicio: failed with 2 bugs and 0 errors  -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

**Reply by DaianaKaneshiro** [reply]
Hola!!! tengo exactamente el mismo error. El test me corre pero el mvn verify muestra los mismos errores

**Reply by gastonprieto** [reply]
Buenas @DaianaKaneshiro y @JuanMartinIwassjuk 

Si es lo mismo fijense que en la primer linea de error dice cual es el problema. También fijense que da  una opción para  ver mas detalles/

>To see bug detail using the Spotbugs GUI, use the following command "mvn spotbugs:gui"

**Reply by gastonprieto** [reply]
(aclaración hay dos errores)

**Reply by JuanMartinIwassjuk** [reply]
> (aclaración hay dos errores)

Hola gaston, gracias por responder, ya pude solucionar los errores que tenia el programa, pero ahora me aparece un error de un plugin que no puede encontrar
![errorjacoco](https://user-images.githubusercontent.com/81983051/165670900-bf8b417c-ae4d-4f68-8090-304c88532e66.png)
pd: diana habla al grupo de dds si no pudiste arreglarlo y te doy una mano

**Reply by JuanMartinIwassjuk** [reply]
tambien busque en esa pagina que linkea pero no pude resolverlo, ahi paso el texto:
```
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  13.926 s
[INFO] Finished at: 2022-04-27T23:52:57-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.spotbugs:spotbugs-maven-plugin:4.2.0:check (Validar Code Smells) on project ejercicio: failed with 2 bugs and 0 errors  -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

**Reply by gastonprieto** [reply]
Fijate lo que dice el error del print `jacoco plugin blah blah` y despues dice algo de coverage. revisate ese mensaje y el foro que eso esta respondido este año

**Reply by JuanMartinIwassjuk** [reply]
listo profe ya lo encontre, muchas graciass

**Reply by DaianaKaneshiro** [reply]
Perdón por no responder, yo aun no he podido resolverlo. Al parecer tengo un error en el código que no logro descubrir y lo cuadriplique...Avisare cuando logre solucionarlo. Muchas gracias.

**Reply by gastonprieto** [reply]
queres armarte otro issue con tu error @DaianaKaneshiro ?

**Reply by DaianaKaneshiro** [reply]
ok. Pueden cerrar este y creare uno nuevo si no logro encontrar los errores en mi código

**Reply by DaianaKaneshiro** [reply]
El error  se encuentra solucionado. Muchas gracias!!
